.jgt_tools.index

```diff
@@ -14,12 +14,12 @@ Default values for each are:
 
 
 JGT Tools - What It Does
-==================================
+========================
 
 .. _api-documentation:
 
 JGT Tools - How To Use It - Module Documentation
-----------------------------------------------------------
+------------------------------------------------
 
 For users of this package, the nitty gritty coding and configuration details.
 
```

Comment:
@bradsbrown and I discussed making the custom bits stored elsewhere (`pyproject.toml`?) and optional and build the `index.rst` dynamically, but felt that would be a FF to this PR.

Reply:
Yeah, in some product specific repos there is a dynamicity to how parts of that file are created.
It's not immediately obvious to me that a `pyproject.toml` file is a good place for embedding `.rst` content...

Reply:
There's a reason I ended that parenthetical with a question mark. The bigger improvement, to me, is to not *require* a user to have an index file; only leverage the ability to customize it, if so desired.